roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -440,6 +440,22 @@ loadbalancer_apiserver_localhost: "{{ loadbalancer_apiserver is not defined }}"
 loadbalancer_apiserver_type: "nginx"
 # applied if only external loadbalancer_apiserver is defined, otherwise ignored
 apiserver_loadbalancer_domain_name: "lb-apiserver.kubernetes.local"
+kube_apiserver_global_address: |-
```

Comment:
Not a great fan of adding those variables here. I'm open to suggestions If someone has a better solution.

Reply:
The thing is that variable does not reference something global, for example if you deploy from a master and you do not have any load balancer It will point to localhost for every hosts (which is not really great). But yeah I'm not a great fan of adding this variable. I can however do something with the filter that you pointed out to reduce It to one variable. WDYT ?